lib/DragNDrop.pm

```diff
@@ -132,7 +126,7 @@ package DragNDrop;
 use strict;
 use warnings;
 
-use Mojo::JSON qw(encode_json);
+use JSON;
```

Comment:
We've switched to Mojo::JSON for this now.  Please switch this back.

Reply:
In general, there are lots of places in this pull request that revert changes that have been made to the upstream code.  @pstaabp pointed out one here, but the new `multicolsWidth` option is removed in this pull request as well.  The `multicols` TeX handling fix is also reverted.  Please check over all of the changes, and make sure only the things that are for this new option are changed.